### DISCUSSION TITLE
Proven working adapter for old aqara devices

### DISCUSSION THREAD

**Discussion by hitokiri8x** [Koenkk/zigbee2mqtt#16423] [Devices]
Hello, I wanted to update my old ( and not so stable ) cc2531 with  a new adapter ( maybe using zigbee 3.0 and future matter support ),  so I buyed a "ZBDongle-E" ( efr32 chip ) and then the nightmare begins: ZHA, Z2M stable & dev versions; they aren't stable devices  not become offline and no longer communicate with coordinator.  
This problem seems related also with skyconnect ( also efr32 ) so I'm wondering  if is a ezsp bug or what..
Has anyone a stable adapter working with:
- [WXKG01LM](https://www.zigbee2mqtt.io/devices/WXKG01LM.html#xiaomi-wxkg01lm)  button 
- [MCCGQ01LM](https://www.zigbee2mqtt.io/devices/MCCGQ01LM.html#xiaomi-mccgq01lm)  window sensor
- [WSDCGQ11LM](https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html#xiaomi-wsdcgq11lm) temp & hum

**Comment by piontec** [comment]
I was using an old CC2531 with my network and everything worked fine, then I decided to "upgrade" to "ZBDongle-e" - worst decision ever, my zigbee network wasn't stable for 1 day since then. Previously, everything worked, including [MCCGQ01LM](https://www.zigbee2mqtt.io/devices/MCCGQ01LM.html#xiaomi-mccgq01lm) now, after deleting all the configs, starting from scratch and only using the latest firmware from sonoff, all my devices except of [MCCGQ01LM](https://www.zigbee2mqtt.io/devices/MCCGQ01LM.html#xiaomi-mccgq01lm) work. This one just disconnects. I think it's time to put dongle-e in a drawer, check back in few months. I tried rcp firmware for simultaneous thread and zigbee - couldn't pair a single device.

**Comment by lux73** [comment]
I can recommend CC2652RB aka slaesh Adapter

Really really stable - plain z2m Installation

**Comment by Shred99** [comment]
"Proven working adapter" is a "how long is a piece of string" question.

My main network has three routers and a TI LAUNCHXL-CC1352P-2 with over 20 Aqara sensors (and a few others).  It's not rock solid, but stability is pretty good considering that some sensors are used to monitor gates and outside sheds that are 20+m from the house.

I have another site with a CC2531 coordinator and 4 x basic Aqara sensor.  That site has been absolutely rock solid for over a year now, despite the old / under powered coordinator.  I guess it's the simplicity of the setup and the fact that the sensors are all inside the house, with the furthest one around 12m from the coordinator with only one plasterboard wall in the way.
